components/chainhook-cli/src/service/mod.rs

```diff
@@ -345,23 +366,187 @@ impl Service {
 pub enum PredicateStatus {
     Scanning(ScanningData),
     Streaming(StreamingData),
-    InitialScanCompleted,
+    InitialScanCompleted(CompletedScanData),
     Interrupted(String),
     Disabled,
 }
 
-#[derive(Debug, Clone, Serialize, Deserialize)]
+#[derive(Debug, Clone, Serialize, Deserialize, Default)]
 pub struct ScanningData {
     pub number_of_blocks_to_scan: u64,
     pub number_of_blocks_scanned: u64,
-    pub number_of_blocks_sent: u64,
```

Comment:
It seemed that Sabby had a misunderstanding of what `number_of_blocks_sent` means, so I thought a rename was in order. I'm open to suggestions other than `number_of_times_triggered`